docs/advanced/node-local-storage.md

```diff
@@ -0,0 +1,358 @@
+# Using Node Local Storage on Kubernetes
+
+This guide shows different ways to use node local storage on Lokomotive, to achieve near
+bare-metal storage performance inside Kubernetes pods.
+
+Before you continue, make sure to read generalities about these concepts:
+ * [Volumes](https://kubernetes.io/docs/concepts/storage/volumes/)
+ * [Persistent Volumes and PVC](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)
+ * [StorageClasses](https://kubernetes.io/docs/concepts/storage/storage-classes/)
+
+More details about specific volumes types used and how to use them on Lokomotive
+will follow, so there is no need to look into volume specifics on the first
+read.
+
+**NOTE**: This has been tested on Packet provider only, some Lokomotive changes may need to be
+ported to other providers for this to work.
+
+## Local Persistent Volumes in Filesystem Mode
+
+This way of using node local storage works by exposing a filesystem mounted in
+the host as a Persistent Volume to pods.
+
+A single Persistent Volume can be bound to only one PVC in ReadWriteOnce access
+mode, so only one pod will be able to use one Persistent Volume in the host.
+However, see the section below to see how to create more than one Persistent
+Volume per node.
+
+The Kubernetes documentation on Local Persistent Volumes using filesystem mode is
+[here](https://kubernetes.io/docs/concepts/storage/volumes/#local) (mixed with
+block mode).
+	
+Bear in mind that this section is about the filesystem mode, not the block mode.
+That will be covered on a different section of this document.
```

Comment:
Do you mean that the block mode will be covered in a different section of this document? If yes, how about rephrasing as:

```
Note that this section is about the filesystem mode, not the block mode.
Information about the block mode will be covered on a different section of this document.
```